l10n/css.json

```diff
@@ -970,6 +970,9 @@
     "ja": "両方の値が length の場合は、length 値として補完されます。両方の値がパーセント値の場合は、パーセント値として補完されます。それ以外の場合は、両方の値が calc() 関数にコンバートされ、length とパーセント値の合計になります (または各値が 0)。そして、これらの <code>calc()</code> 関数は、それぞれ半分ずつ補完された実数を持ちます",
     "ru": " когда оба значения являются длинами, они интерполируются как длины; когда оба значения являются процентами, они интерполируются как проценты; в остальных случаях, оба значения преобразуются функцией <code>calc()</code> в сумму длины и процента (каждый из них может быть равен нулю), а затем эта функция интерполирует каждую половину как вещественные числа."
   },
+  "mapToRange0To1": {
```

Comment:
Since I added a new entry to the schema, I decided to add a corresponding `l10n` entry for `en-US`. It's possible that this is superfluous, since it does not add localization beyond the base language for MDN. 

My rationale for adding it anyway:

- It helps document `mapToRange0To1` with a human readable version
- It makes it more accessible for others to see where translations are missing and add their native language